internal/testhelper/testhelper.go

```diff
@@ -81,9 +85,15 @@ func SetupDeps(sensors []string) resource.Dependencies {
 	return deps
 }
 
-func getGoodLidar() *inject.Camera {
+func getValidSensor(isReplay bool) *inject.Camera {
```

Comment:
[opt] I think it'd be better to follow the pattern of a separate helper (`getReplaySensor`) than to pass a bool in, especially since the duplicated logic would be very straightforward and is also duplicated in the other helpers

Reply:
SGTM, made that change!

Reply:
ty!